projects/Unit/Unit.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
     <AssemblyOriginatorKeyFile>../rabbit.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
+    <RunAnalyzersDuringBuild>false</RunAnalyzersDuringBuild>
```

Comment:
@danielmarbach why is this required? I see that no analyzers are installed in this project, nor in the parent `Directory.Build.targets`. Or is there a transitive dependency somewhere?

Reply:
Without the flag in VS

![image](https://user-images.githubusercontent.com/174258/93250269-dfc90300-f792-11ea-97d2-1feff8a1fc32.png)

Reply:
It is related to `PrivateAssets="None"`. If switched to `All` we can remove the exclusion from analysis

Reply:
Raised https://github.com/rabbitmq/rabbitmq-dotnet-client/pull/941

Reply:
> `PrivateAssets="None"`

👍 yes, that explains it.